### PR TITLE
A0-932 Added follow-up finalization check to some e2e tests

### PR DESCRIPTION
**Pull request by Marcin-Radecki**
# Description

Please include a summary of the change and which issue is fixed. Please also include relevant motivation and context. List any dependencies that are required for this change.

## Type of change

Please delete options that are not relevant.

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

# Checklist:

- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] Infrastructure updated accordingly
- [ ] I have made corresponding changes to the documentation
- [ ] New documentation created
- [ ] Bump `spec_version` and `transaction_version` if relevant
- [ ] Bump `aleph-client` version if relevant

### CODE REVIEW

File: .github/actions/README.md [hunk 1]
```diff
@@ -47,7 +47,8 @@ Stops sccache server. Use together with `restore-cache`.
 ---
 
 ## `run-e2e-test`
-This action runs a single test from the e2e test suite. It requires a test case, which is the name of the test.
+his action runs a single test from the e2e test suite. It requires a test case, which is the name of the test. It runs
```
Comment (pmikolajczyk41) [inline .github/actions/README.md:50]: typo: missing letter `T` :)

**Review by pmikolajczyk41 — APPROVED** [review]

File: .github/actions/README.md (hunk 1 above)
Comment (Marcin-Radecki) [reply .github/actions/README.md:50]: Thanks for noticing

File: .github/actions/README.md [hunk 2]
```diff
@@ -47,7 +47,8 @@ Stops sccache server. Use together with `restore-cache`.
 ---
 
 ## `run-e2e-test`
-This action runs a single test from the e2e test suite. It requires a test case, which is the name of the test.
+This action runs a single test from the e2e test suite. It requires a test case, which is the name of the test. It runs
```
Comment (maciejzelaszczyk) [inline .github/actions/README.md:50]: Would change the last sentence of the comment to:
_It optionally runs the finalization e2e testcase, which is helpful after some e2e tests to double-check nothing is broken._

**Review by maciejzelaszczyk — CHANGES_REQUESTED** [review]
One nit.

File: .github/actions/README.md (hunk 2 above)
Comment (Marcin-Radecki) [reply .github/actions/README.md:50]: done

**Review by maciejzelaszczyk — APPROVED** [review]

**Pull request merged**